.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+##### Motivation for the changes
+
+<!-- An issue reference or explanation why the change is desirable. -->
+<!-- This speeds up step 1 of the process. -->
+
+
+
+
+
+
+##### Checklist for maintainers
+
+<!-- Contributors: please leave this as is -->
+<!-- Maintainers: tick if completed or explain if not relevant -->
+
+ - [ ] is the idea good? has it been discussed by the Nix team?
+ - [ ] unit tests
+ - [ ] functional tests (`tests/**.sh`)
+ - [ ] documentation in the manual
```

Comment:
```suggestion
 - [ ] user-facing documentation is appropriate
```

We may want to be more precise about what "appropriate" means, maybe there is a better word...

Reply:
It's a thing of balance, neither too verbose, nor too terse. It's hard to capture where to strike that balance without really having a clearly defined "user/reader persona" for which one is supposed to optimize.

It's a problem, indeed. "balanced" suffers from the same imprecision, but better captures that it's a judgemental trade off.

Reply:
I don't understand why this is an improvement.
It seems a bit wordy and we already have "tick if completed or explain if not relevant".

Reply:
With the additional information in the other notes its indeed good as it is.